BFS/ccconac/7569.js

```diff
@@ -0,0 +1,66 @@
+const input = require('fs')
+  .readFileSync('text.txt')
+  .toString()
+  .trim()
+  .split('\n');
+
+const [M, N, H] = input.shift().split(' ').map(Number);
+
+const dx = [-1, 1, 0, 0, 0, 0];
+const dy = [0, 0, -1, 1, 0, 0];
+const dz = [0, 0, 0, 0, -1, 1];
+const queue = [];
+
+const boxes = Array.from(
+  Array(H),
+  () => Array.from(N),
+  () => Array.from(Array(M).fill(0))
+);
```

Comment:
이 부분 실행해봤는데 3차원 배열이 제대로 만들어지지 않아요!
```suggestion
const boxes = Array.from({ length: H }, ()=>{
   Array.from({ length:N }, ()=>{
      Array(M).fill(0)
   }
})
```